templates/stake-pool-template/move/tests/test_end_to_end.move

```diff
@@ -114,6 +117,11 @@ module stake_pool_addr::test_end_to_end {
         staker2 reward index is 29, last claim ts is 100, stake = 30000, claimed reward = 30000 * (29 - 20) = 2700
         */
 
+        {
+            let (_, _, _, unique_stakers) = stake_pool::get_stake_pool_data();
+            assert!(unique_stakers == 0, unique_stakers);
+        };
+
         // let (
         //     user_staked_amount,
```

Comment:
can we delete these comments?

Reply:
my original intention is to make it easier for developers to debug if they want, because i always need to lookup how to debug print in move

Reply:
i uncommented and cleaned it up